scantext/templates/scantext/includes/license_summary_level.html

```diff
@@ -0,0 +1,35 @@
+{% load humanize %}
+<nav class="level is-mobile">
+  <div class="level-item has-text-centered">
+    <div>
+      <p class="heading">Percentage Of License Text</p>
+      <p class="title">
+          <span>
+            {{ expr.percentage_of_license_text }}
+          </span>
+      </p>
+    </div>
+  </div>
+  <div class="level-item has-text-centered">
+    <div>
+      <p class="heading">License Expressions</p>
+      <p class="title">
+          <span>
+            {{ expr.license_expressions | length }}
```

Comment:
Spaces around "|" are not consistent with others existing templates.

Reply:
yes, will correct them in the upcoming commit